mne/viz/topo.py

```diff
@@ -72,6 +75,12 @@ def iter_topography(
         If True, an additional axis is created in the bottom right corner
         that can be used to, e.g., construct a legend. The index of this
         axis will be -1.
+    select : bool
+        Whether to enable the lasso-selection tool to enable the user to select
+        channels. The selected channels will be available in
+        ``fig.lasso.selection``.
```

Comment:
I wonder if it makes sense to also do a `logger.info()` each time the user does a lasso? Or does that seem like overkill?  Might be more convenient than having to query `fig.lasso.selection` each time.

Reply:
I like this idea.